src/core/zowe/core_for_zowe_sdk/profile_manager.py

```diff
@@ -273,12 +279,29 @@ def load(
 
             missing_secure_props.extend(profile_loaded.missing_secure_props)
 
-            if override_with_env:
-                env_var = {**self.get_env(cfg)}
 
             if i == 1 and profile_props:
                 break  # Skip loading from global config if profile was found in project config
 
+        usrProject = self.project_user_config.profiles
+        project = self.project_config.profiles
+
+        # Creating copies of the usrProject and project objects
+        usrProjectCopy = deepcopy(usrProject)
+        projectCopy = deepcopy(project)
+        prjt = always_merger.merge(usrProject, project)
+
+        usrGlobal = self.global_user_config.profiles
+        glbal = self.global_config.profiles
+        
+        # Creating copies of the usrGlobal and glbal objects
+        usrGlobalCopy = deepcopy(usrGlobal)
+        glbalCopy = deepcopy(glbal)
```

Comment:
I have created the copies using `deepcopy` for the objects before getting merged